lib/currency-locales/locales/th.yml

```diff
@@ -1,23 +1,47 @@
+---
 th:
   number:
     currency:
       format:
         delimiter: ","
-        format: "%u %n"
+        format: "%n %u"
```

Comment:
I'd have to analyze the kind of space but it changed -- probably thin nbsp

Reply:
No space